docs/deploy.md

```diff
@@ -106,4 +106,60 @@ APP_SECRET=XXXXXXXX-XXXX-XXXX-XXXX-XXXXXXXXXXXX hathora dev --only client
 10. To destory your application run `hathora cloud destroy --appName yourAppName`
 
 
-Self-hosting will remain an option for those that want more control.
\ No newline at end of file
+Self-hosting will remain an option for those that want more control.
+
+## Setting up Continuous Delivery with Github Actions
+
+Here you will learn how to set up continuous delivery with Github Actions but you can use the platform of your choice. 
+
+In your Github Repo, select the actions tab and create a custom `.yml` file. Remove the auto-generated code and replace with:
```

Comment:
you can include something like:

"you can name the file whatever you want, for example `deploy.yml`